Update README.md (#2361)
clockworklabs/SpacetimeDB: README.md

SpacetimeDB is an embedded database library that you can use to extend your own applications. You can also run SpacetimeDB as a standalone database server via the `spacetime` CLI tool.

> You can install and run the `spacetime` CLI tool via Cargo, Docker, or by downloading the binary directly from our website.

Reviewer: ```suggestion
You can install and run the `spacetime` CLI tool via Cargo, Docker, or by running the installation instructions on our website: https://spacetimedb.com/install
```

#### Install on macOS

Installing SpacetimeDB on macOS is as easy as downloading the binary and running it. You can also install on macOS using the Docker instructions below.

```bash

> curl --proto '=https' --tlsv1.2 -sSf https://install.spacetimedb.com | sh

Reviewer: I think this is a little more brittle than just pointing people to our install page but I'm not opposed

Author: Let's just roll it since it's not more broken.
